libs/partners/huggingface/langchain_huggingface/llms/huggingface_pipeline.py

```diff
@@ -55,7 +55,7 @@ class HuggingFacePipeline(BaseLLM):
     """
 
     pipeline: Any = None  #: :meta private:
-    model_id: str = DEFAULT_MODEL_ID
+    model_id: Optional[str] = None
     """Model name to use."""
```

Comment:
Could you update the doc-string to explain what the behavior is?

Reply:
@eyurtsev We have updated the doc-string